content/_includes/contact.macros.njk

```diff
@@ -62,11 +62,6 @@ params:
     required=true
   ) }}
 
-  <label data-honey aria-hidden="true">
-    <span>Don't fill out this field:</span>
-    <input name="subject" />
-  </label>
```

Comment:
If we want to remove this, we can also remove the `netlify-honeypot` attr from the `<form>` above.